### ISSUE TITLE
Stop training early if accuracy is 100%

### ISSUE THREAD

**Issue by stweil** [mittagessen/kraken#519]
Currently the training process continues even if the validation reports 100% accuracy although no further improvement can be expected of course. I observed a recognition training in eScriptorium where 100% was already reached after the first epoch, but the process continued with 10 more useless epochs, some of them reaching 100%, too.

**Comment by mittagessen** [comment]
Hmm, I suspect this is a bug in pytorch-lightning as we set an upper abort threshold in the early stopping already. Can you tell me which pytorch-lightning version you're running under? I'm currently on vacation but can take a look next week.

**Comment by stweil** [comment]
It's running with pytorch-lightning 2.0.2.
